### PR TITLE
Docs: fix update scripts usage

### PR DESCRIPTION
**Pull request by nnsW3**
Hello
I fixed several spelling issues.
Br, Elias.

### CODE REVIEW

File: docs/options.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ test-requires = "pytest"
 test-command = "pytest {project}/tests"
 ```
 
-The complete set of defaults for the current version of cibuildwheel are shown below:
+The complete set of defaults for the current version of cibuildwheel is shown below:
```
Comment (joerick) [inline docs/options.md:107]: the previous wording is correct, defaults are plural.

**Comment by nnsW3** [comment]
done

**Review by henryiii — APPROVED** [review]

File: docs/contributing.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ nox -s docs                    # Build and serve the documentation
 nox -s build                   # Make SDist and wheel
 ```
 
-More advanced users can run the update scripts. `update_pins` should work directly, but `update_constraints` needs all versions of Python installed. If you don't want to do that locally, a fast way to run it to use docker to run nox:
+More advanced users can run the update scripts. `update_pins` should work directly, but `update_constraints` needs all versions of Python installed. If you don't want to do that locally, a fast way to run it is to use docker to run nox:
```
Comment (mayeut) [inline docs/contributing.md:116]:
```suggestion
More advanced users can run the update scripts. `nox -s update_pins update_constraints`.
```

The `update_constraints` comment is most likely obsolete now that constraints are updated with `uv`.
The mention of `docker run ...` should be removed.

File: docs/contributing.md [hunk 3]
```diff
@@ -113,10 +113,10 @@ nox -s docs                    # Build and serve the documentation
 nox -s build                   # Make SDist and wheel
 ```
 
-More advanced users can run the update scripts. `update_pins` should work directly, but `update_constraints` needs all versions of Python installed. If you don't want to do that locally, a fast way to run it to use docker to run nox:
+More advanced users can run the update scripts. `nox -s update_pins update_constraints`.
 
 ```console
-docker run --rm -itv $PWD:/src -w /src quay.io/pypa/manylinux_2_24_x86_64:latest pipx run nox -s update_constraints
+nox -s update_constraints
```
Comment (mayeut) [inline docs/contributing.md:119]:
```suggestion
nox -s update_constraints # update all constraints files in cibuildwheel/resources
nox -s update_pins        # update tools, python interpreters & docker images used by cibuildwheel
```

File: docs/contributing.md [hunk 4]
```diff
@@ -113,10 +113,10 @@ nox -s docs                    # Build and serve the documentation
 nox -s build                   # Make SDist and wheel
 ```
 
-More advanced users can run the update scripts. `update_pins` should work directly, but `update_constraints` needs all versions of Python installed. If you don't want to do that locally, a fast way to run it to use docker to run nox:
+More advanced users can run the update scripts. `nox -s update_pins update_constraints`.
```
Comment (mayeut) [inline docs/contributing.md:116]:
```suggestion
More advanced users can run the update scripts:
```

**Pull request merged**